scripts/fast-start.sh

```diff
@@ -102,12 +109,25 @@ fi
 # Let's make sure we got the IPs to use in the configuration.
 case "$PROVIDER" in 
 "AWS" )
-    # We have already discovered them.
+    # Set variables for AWS.
+    ADMIN_PASSWD="$(ec2metadata --instance-id)"
+    ADMIN_EMAIL="a@a.com"
+    ;;
+"GCE" )
+    # We assume a single interface here.
+    PUBLIC_IP="$(wget -O - --header 'Metadata-Flavor: Google' -q http://169.254.169.254/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip)"
```

Comment:
Make 'http://169.254.169.254/computeMetadata/v1/instance/' a constant since you're using it in multiple places.